src/main/webapp/schema/schema.json

```diff
@@ -1300,6 +1300,7 @@
           "$ref": "#/definitions/DashboardPage"
         }
       },
+      "required": ["configuration", "inputs"],
```

Comment:
We probably stick to only configuration page being required. Some people create add-ons which have only configuration page, there is a use case like this.